src/components/body/content/categories/index.scss

```diff
@@ -0,0 +1,30 @@
+.wrapper__categories {
+    width: 1376px;
+    height: 828px;
+}
+
+.categories {
+    align-items: flex-start;
```

Comment:
Do you use here grid or flex? It looks like a mess...

align-items: flex-start;
...
display: grid;

Reply:
this using grid)